### ISSUE TITLE
Footnote inside of span class fails

### ISSUE THREAD

**Issue by SamWhited** [davidfstr/rdiscount#121]
Using version `2.0.7.3` with the footnotes extension enabled, the following fails to produce the footnote (the link appears, and the span is output correctly with the class but no footnote is actually generated at the bottom of the page):

```markdown
[Some statement[^footnote].](class:someclass)

[^footnote]: Some footnote
```

However, adding the span manually works as expected (there is a footnote at the bottom of the output):

```markdown
<span class="someclass">
Some statement[^footnote].
</span>

[^footnote]: Some footnote
```

**Comment by SamWhited** [comment]
Fixed upstream in 46e147599fafe72f4ea694c122b540d1a766ee96

**Comment by davidfstr** [comment]
Cool. I'll integrate this once there's a new Discount release.

**Comment by davidfstr** [comment]
Fixed in RDiscount 2.1.8, which is now released.
